consumer.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"fmt"
 
-	"github.com/sarama"
+	"github.com/Shopify/sarama"
```

Comment:
Import is often subjective to the user's local setup. The current part matches what you would get if you did `go get github.com/Shopify/sarama`. I would advice we leave it that way.

Reply:
imho, for the standard convention, it would be `github.com/Shopify/sarama` , isn't it ? `github.com/sarama` gives me a problem because the lib can't be found for that path. by changing it to `github.com/Shopify/sarama` , people can get the project and directly run it. cmiiw

Reply:
Ahh, you are right. My file path was wrong at the time. It should be `github.com/Shopify/sarama` 👍